doc/coding-conventions.xml

```diff
@@ -653,15 +727,19 @@ args.stdenv.mkDerivation (args // {
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>If it’s a <emphasis>game</emphasis>:</term>
+     <term>
+      If it’s a <emphasis>game</emphasis>:
+     </term>
      <listitem>
       <para>
        <filename>games</filename>
       </para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Else:</term>
+     <term>
+      Else:
+     </term>
      <listitem>
       <para>
        <filename>misc</filename>
```

Comment:
I sort of think none of these should be `<term>`. What do you think?

Reply:
That if they are or aren't should be fixed in another PR :).

Though, it seems right in how it may be abusing the `variablelist` for its appearance instead of its meaning. Except `variablelist` *could* be more apt if instead of presenting it as a list of choices, it was more a description of the categories.

[`variablelist` from TDG](https://tdg.docbook.org/tdg/5.2/variablelist.html):

>  A list in which each entry is composed of a set of one or more terms and an associated description.

Doesn't make me think it's "programming" variables only. Looks like it's a catchall "term / definition" list. And it does look like `term` is the only node in there that isn't the definition.

So (while this may be wrong WRT valid docbook) this may be turned outside in this way:

```
        <term>
          <filename>development/interpreters</filename>        </term>
        <listitem>
         <para>
         If it’s an <emphasis>interpreter</emphasis>.
         (e.g. <filename>guile</filename>)
         </para>
        </listitem>
```